README.md

```diff
@@ -58,6 +58,13 @@ To install xpk, run the following command:
 pip install xpk
 ```
 
+If you are running XPK by cloning GitHub repository, first run the
+following command to begin using XPK commands:
+
+```shell
+pip install cloud-accelerator-diagnostics
```

Comment:
We can suggest installing all dependencies in case we add more in the future.

```
git clone https://github.com/google/xpk.git
# Install dependencies such as cloud-accelerator-diagnostics
pip install .
```

Also want to add a section about:

```
If you see an error saying: `This environment is externally managed`. Please use a virtual environment:

Example:

```shell
  ## One time step of creating the venv
  VENV_DIR=~/venvp3
  python3 -m venv $VENV_DIR
  ## Enter your venv.
  source $VENV_DIR/bin/activate
  ## Same steps for cloning and installing dependencies.
```